### PR TITLE
Add IsFinal field to the WebUrl DB class and remove Default field from the WebTemplate

### PR DESCRIPTION
**Pull request by un-tone**
For https://github.com/StarcounterPrefabs/Website/issues/41

### CODE REVIEW

**Comment by un-tone** [comment]
Related to the PR https://github.com/StarcounterPrefabs/Website/pull/43

File: Ring6/Web/WebTemplate.cs [hunk 1]
```diff
@@ -4,8 +4,6 @@
 namespace Simplified.Ring6 {
     [Database]
     public class WebTemplate : Something {
-        public bool Default;
```
Comment (warpech) [inline Ring6/Web/WebTemplate.cs]: To make the app upgrade smooth, we do not remove fields in Simplified but mark them as `Obsolete`. See: https://github.com/StarcounterPrefabs/UserAdmin/pull/49

Comment (un-tone) [inline Ring6/Web/WebTemplate.cs]: Sure, I missed it. Fixed.

**Comment by un-tone** [comment]
I will create another PR for this changes

**Pull request closed without merging**